### ISSUE TITLE
scrolling_spectro() function fails to produce mp4

### ISSUE THREAD

**Issue by cebus-imitator** [maRce10/dynaSpec#14]
I ran the example code:

```
library(dynaSpec)
library(viridis)
library(tuneR)
library(seewave)
library(warbleR)  

data("canyon_wren")
scrolling_spectro(
	wave = canyon_wren,
	wl = 300,
	t.display = 1.7,
	pal = viridis,
	grid = FALSE,
	flim = c(1, 9),
	width = 1000,
	height = 500,
	res = 120,
	file.name = "default.mp4"
)
```
but get the following:

 |++++++++++++++++++++++++++++++++++++++++++++++++++| 100% elapsed=03m 19s
Error in system(cll1, intern = TRUE, ignore.stdout = TRUE, ignore.stderr = TRUE) : 
  error in running command
Warning messages:
1: In graphics::par(opar) : graphical parameter "cin" cannot be set
2: In graphics::par(opar) : graphical parameter "cra" cannot be set
3: In graphics::par(opar) : graphical parameter "csi" cannot be set
4: In graphics::par(opar) : graphical parameter "cxy" cannot be set
5: In graphics::par(opar) : graphical parameter "din" cannot be set
6: In graphics::par(opar) : calling par(new=TRUE) with no plot
7: In graphics::par(opar) : graphical parameter "page" cannot be set

I used:
macOS Monterey Version 12.4
R version 4.1.3 (2022-03-10) -- "One Push-Up"
RStudio version: RStudio 2022.02.3+492 "Prairie Trillium" Release (1db809b8323ba0a87c148d16eb84efe39a8e7785, 2022-05-16) for macOS
Mozilla/5.0 (Macintosh; Intel Mac OS X 12_4_0) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.10 Chrome/69.0.3497.128 Safari/537.36

If try to run a wav file locally from my own computer, I get a different error:

Error in z[(fl1:fl2) + 1, ] : subscript out of bounds
Warning messages:
1: In graphics::par(opar) : graphical parameter "cin" cannot be set
2: In graphics::par(opar) : graphical parameter "cra" cannot be set
3: In graphics::par(opar) : graphical parameter "csi" cannot be set
4: In graphics::par(opar) : graphical parameter "cxy" cannot be set
5: In graphics::par(opar) : graphical parameter "din" cannot be set
6: In graphics::par(opar) : calling par(new=TRUE) with no plot
7: In graphics::par(opar) : graphical parameter "page" cannot be set

**Comment by cebus-imitator** [comment]
Thank you for looking into this. The scrolling_spectro() function looks like it will produce exactly what I need. So, I hope I can get it to work on my computer.

in terminal `brew upgrade ffmpeg`
Warning: ffmpeg 5.0.1_2 already installed

I also tried it using Parallels on my Mac to run RStudio in Windows 10s. I downloaded and extracted the files and moved them to the C drive.
C:\ffmpeg\bin

Still no luck when running the code. I am not sure what I am doing wrong with the ffmpeg application.
